htdocs/expedition/class/expedition.class.php

```diff
@@ -84,11 +84,84 @@ class Expedition extends CommonObject
+		'fk_shipping_method' => ['type' => 'integer',      'label' => 'ShippingMethod', 'enabled' => 1,                          'position' => 170, 'notnull' => 0, 'visible' => -1, 'index' => 1, 'default' => 0],
+		'tracking_number'    => ['type' => 'varchar(50)',  'label' => 'TrackingNumber', 'enabled' => 1,                          'position' => 180, 'notnull' => 0, 'visible' => -1],
+		'fk_statut'          => ['type' => 'smallint(6)',  'label' => 'Status',         'enabled' => 1,                          'position' => 190, 'notnull' => 0, 'visible' => -1, 'index' => 1, 'default' => 0],
+		'billed'             => ['type' => 'smallint(6)',  'label' => 'Billed',         'enabled' => 1,                          'position' => 200, 'notnull' => 0, 'visible' => -1],
+		'height'             => ['type' => 'float',        'label' => 'Height',         'enabled' => 1,                          'position' => 210, 'notnull' => 0, 'visible' => -1],
+		'width'              => ['type' => 'float',        'label' => 'Width',          'enabled' => 1,                          'position' => 220, 'notnull' => 0, 'visible' => -1],
+		'size_units'         => ['type' => 'integer',      'label' => 'SizeUnits',      'enabled' => 1,                          'position' => 230, 'notnull' => 0, 'visible' => -1],
+		'size'               => ['type' => 'float',        'label' => 'Size',           'enabled' => 1,                          'position' => 240, 'notnull' => 0, 'visible' => -1],
+		'weight_units'       => ['type' => 'integer',      'label' => 'WeightUnits',    'enabled' => 1,                          'position' => 250, 'notnull' => 0, 'visible' => -1],
+		'weight'             => ['type' => 'float',        'label' => 'Weight',         'enabled' => 1,                          'position' => 260, 'notnull' => 0, 'visible' => -1],
+		'note_private'       => ['type' => 'text',         'label' => 'NotePrivate',    'enabled' => 1,                          'position' => 270, 'notnull' => 0, 'visible' => -1],
+		'note_public'        => ['type' => 'text',         'label' => 'NotePublic',     'enabled' => 1,                          'position' => 280, 'notnull' => 0, 'visible' => -1],
+		'model_pdf'          => ['type' => 'varchar(255)', 'label' => 'PDFTemplate',    'enabled' => 1,                          'position' => 290, 'notnull' => 0, 'visible' => -1],
+		'last_main_doc'      => ['type' => 'varchar(255)', 'label' => 'LastMainDoc',    'enabled' => 1,                          'position' => 300, 'notnull' => 0, 'visible' => -1],
+		'fk_incoterms'       => ['type' => 'integer',      'label' => 'IncotermCode',   'enabled' => '$conf->incoterm->enabled', 'position' => 310, 'notnull' => 0, 'visible' => -1],
+		'location_incoterms' => ['type' => 'varchar(255)', 'label' => 'IncotermLabel',  'enabled' => '$conf->incoterm->enabled', 'position' => 320, 'notnull' => 0, 'visible' => -1],
```

Comment:
make use of isModEnabled instead $conf->incoterm->enabled

Reply:
and with the fix suggested by @mdeweerd 
![image](https://github.com/user-attachments/assets/ca77325e-fa79-44a2-a351-37a113b8bae4)